config/default.yaml

```diff
@@ -2,6 +2,10 @@ server:
   port: 5999
   appRoot: "/bucky"
 
+handlers:
+  http_headers:
```

Comment:
I believe the rest of this file uses camel case, so this should be `httpHeaders` or `HTTPHeaders`.  Alternatively you could create an option for just `allowOrigin`, and just apply that single header.